test/integration/targets/connection_aws_ssm/runme.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+# Setup Environment
```

Comment:
Add `set -eux` to the start of shell scripts.

Reply:
Added `set -eux` in https://github.com/ansible/ansible/pull/49652/commits/d69abb2abcc0c00912ae786c9f2a81bf489073a7